recipes/platform.equality/all/test_package/conanfile.py

```diff
@@ -0,0 +1,20 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def requirements(self):
+        self.requires("gtest/cci.20210126")
```

Comment:
Please reduce the test_package we lean against writing test code for the library.
Include headers and make an object to test it links 

This is a little on the complicated side